.github/scripts/artifact-upload-params.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+case "$ARTIFACT_SELECTOR" in
+  peer)
```

Comment:
I think it would make more sense for this to be the default selector mode for this file, and not leak beyond it; then we don't change the semantics for the existing CI jobs (which we arguable shouldn't do here)